docs/changes.rst

```diff
@@ -31,6 +31,17 @@ Released: not yet
 
 **Enhancements:**
 
+* Extend MOFWBEMConnection to generate an exception if the compile of a
+  class with reference parameters or properties reference class is not in the
+  repository or if the class defined for an EmbeddedInstance qualifier is
+  not in the repository or. This makes use of the capability with the
+  mof compiler (extended slightly) to search the search path defined for the
+  compile for the corresponding missing classes if they are not in the
+  repository.  This means that the mof_compiler can be used to create complete
+  class repository builds without having to define specifically all dependent
+  classes for the classes defined for the build if those classes are in the
+  search path. See issue # 1160
```

Comment:
Please remove the space between "#" and the issue number. That allows for better searching.

Reply:
Fixed and pushed this fix.  Found another place with same issue